SQL/SQL (basic).sublime-syntax

```diff
@@ -445,6 +450,11 @@ contexts:
       scope: storage.modifier.sql
     - include: expressions-or-column-name
 
+  use-db:
+    - match: \b(?i:use)\b
+      scope: keyword.context.tsql
+      push: [database-name, single-identifier-after-whitespace]
```

Comment:
Maybe rather?

```yaml
      push:
        - database-name
        - single-identifier-after-whitespace
```

It's easier to (1) see the depth and (2) read/review future diffs.

Reply:
fixed, thanks for pointing it out. It is going to take me a while to get used to this style, maybe I'll write myself a linter ;)